docs/source/modeling-transactions.rst

```diff
@@ -58,23 +73,9 @@ The output of ``T2`` requires the parameter ``x`` to be a signature that matches
 The key is declared at line 1, and it is prefixed by the keyword ``wif:`` to denote that it is expressed in the Wallet Import Format [#f1]_.
```

Comment:
Typo:
i.e. ``T1@1`` and ``T1@2`` respectively ->
i.e. ``T1@0`` and ``T1@1`` respectively